src/tests/final_tests.spec.js

```diff
@@ -0,0 +1,63 @@
+const { expect } = require('@playwright/test');
+const { test } = require('../fixture');
+import { sortItems, calculateRandomIndexItems} from '../helper_func';
+import {filterOptions} from '../test_data';
+import {userData, userCredentials} from '../user_data';
+
+test.describe('Verify sorting and adding products', () => {
+    test.beforeEach(async ({ loginPage}) => {
+        const user = userCredentials;
```

Comment:
do we need new variable for it. we can access it straight forward from userCredentials